zboss/include/zboss_api_core.h

```diff
@@ -49,6 +49,25 @@
 #include "zb_types.h"
 #include "zb_channel_page.h"
 #include "zb_errors.h"
+
+/* zb_callback_t is used in osif if we have serialk API */
```

Comment:
Thanks for pointing that out! I will create a PR to ZOI to update this. I do not consider it as a blocker for this PR, since fixing spelling mistakes is a different story than adding necessary scheduler API changes.

Reply:
PR 60 created.